internal/fs/hash_test.go

```diff
@@ -0,0 +1,40 @@
+package fs
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestHashFromNodeWithFile(t *testing.T) {
+	actual, err := HashFromNode("", "./testdata/blob")
+	if err != nil {
+		t.Fatal(err)
+	}
+	expected := "9ccd71eec554488c99eb9205b6707cb70379e1aa637faad58ac875278786f2ff"
+	if actual != expected {
```

Comment:
we also go a little further in gps by using a convention of:

```
(GOT): <data>
(WNT): <data>
```

which helps a bit with scannability. not that it matters with hash digests, ofc - just, again, consistency.